Code-For-All/src/HomeComponents/Header.jsx

```diff
@@ -1,74 +1,151 @@
+import { useNavigate } from "react-router-dom";
+import React, { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBars, faXmark } from "@fortawesome/free-solid-svg-icons";
 import logo from "../assets/logo.png";
-import React, { useState } from 'react';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faBars, faXmark } from '@fortawesome/free-solid-svg-icons';
 
 const scrollToSection = (elementId) => {
+  if (elementId == null) {
```

Comment:
just a question, this fixes the nav bar not scrolling to the correct page when we are on the leader board page correct?